parachain/primitives/router/src/inbound/mod.rs

```diff
@@ -37,6 +37,8 @@ pub enum Command {
 	RegisterToken {
 		/// The address of the gateway
 		gateway: H160,
+		/// The address of the owner
+		owner: H160,
```

Comment:
Let's use more specific naming. All tokens are really always owned by a single agent
```suggestion
		agent: H160,
```